proto/umee/lend.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+package umeenetwork.umee.umee;
+
+option go_package = "github.com/umee-network/umee/x/umee/types";
+
+// Generated when lender attempts to lend base assets to module
+message MsgLendAsset {
+  string denom = 1;
+  int64 amount = 2;
+  // Question: Does the lender address need to be part of the message?
```

Comment:
Comment contains question for @alexanderbez

Reply:
@toteki yes, we need a field -- `string lender = 3;`

Reply:
Also, the fields are wrong. It should be:

```proto
import "cosmos/base/v1beta1/coin.proto";
import "gogoproto/gogo.proto";

message MsgLendAsset {
  string lender = 1;
  cosmos.base.v1beta1.Coin amount  = 2 [(gogoproto.nullable) = false];
}
```